### DISCUSSION TITLE
Padding is invalid and cannot be removed.

### DISCUSSION THREAD

**Discussion by zwluoqi** [pixiv/vroid-sdk-developers#58] [Q&A]
Envrionment：Windows11
SDK Version ：0.12.0
Use CharacterModelExample Demo To Test
when load model ,it produce a error:
![image](https://user-images.githubusercontent.com/3363485/204730610-ceae8b65-0222-4544-be53-0795ec8c7d3c.png)
![image](https://user-images.githubusercontent.com/3363485/204731046-a6c76e8e-49fc-4b69-9b86-0497f2de6a1f.png)

`
Padding is invalid and cannot be removed.
UnityEngine.Debug:LogError (object)
VRoidSDK.Examples.Core.Renderer.ApiErrorRenderer:.ctor (VRoidSDK.Examples.Core.Model.ApplicationModel) (at Assets/VRoidSDK/Examples/Core/Renderer/ApiErrorRenderer.cs:14)
VRoidSDK.Examples.CharacterModelExample.Controller.CharacterModelDetailController/<>c__DisplayClass8_0:<UseCharacterModel>b__3 (Pixiv.VroidSdk.ModelLoadFailException) (at Assets/VRoidSDK/Examples/CharacterModelExample/Scripts/Controller/CharacterModelDetailController.cs:94)
Pixiv.VroidSdk.ModelLoader/<>c__DisplayClass5_0:<LoadVrmData>b__0 (Pixiv.VroidSdk.Api.DataModel.ApiErrorFormat) (at Assets/VRoidSDK/Plugins/Pixiv/VroidSdk/ModelLoader.cs:89)
Pixiv.VroidSdk.Cache.ModelDataCache:Fetch (Pixiv.VroidSdk.Api.DataModel.CharacterModel,System.Action`1<byte[]>,System.Action`1<single>,System.Action`1<Pixiv.VroidSdk.Api.DataModel.ApiErrorFormat>)
Pixiv.VroidSdk.ModelLoader:LoadVrmData (Pixiv.VroidSdk.Api.DataModel.CharacterModel,System.Action`1<byte[]>,System.Action`1<single>,System.Action`1<Pixiv.VroidSdk.ModelLoadFailException>,System.Func`2<VRM.VRMData, VRM.VRMImporterContext>) (at Assets/VRoidSDK/Plugins/Pixiv/VroidSdk/ModelLoader.cs:86)
VRoidSDK.Examples.CharacterModelExample.Controller.CharacterModelDetailController/<>c__DisplayClass8_0:<UseCharacterModel>b__0 (Pixiv.VroidSdk.Api.DataModel.Account) (at Assets/VRoidSDK/Examples/CharacterModelExample/Scripts/Controller/CharacterModelDetailController.cs:79)
VRoidSDK.Examples.Core.Controller.ApiController/<>c__DisplayClass14_0:<LoginToVroidHub>b__0 (Pixiv.VroidSdk.Api.DataModel.Account) (at Assets/VRoidSDK/Examples/Core/Controller/ApiController.cs:103)
VRoidSDK.Examples.Core.Controller.ApiController:GetAccountInfo (System.Action`1<Pixiv.VroidSdk.Api.DataModel.Account>,System.Action`1<Pixiv.VroidSdk.Api.DataModel.ApiErrorFormat>) (at Assets/VRoidSDK/Examples/Core/Controller/ApiController.cs:193)
VRoidSDK.Examples.Core.Controller.ApiController:LoginToVroidHub (System.Action`1<VRoidSDK.Examples.Core.Renderer.IRenderer>,System.Action`1<Pixiv.VroidSdk.Api.DataModel.Account>) (at Assets/VRoidSDK/Examples/Core/Controller/ApiController.cs:99)
VRoidSDK.Examples.Core.Controller.BaseController:CheckLogin (VRoidSDK.Examples.Core.Controller.ApiController,System.Action`1<VRoidSDK.Examples.Core.Renderer.IRenderer>,System.Action`1<Pixiv.VroidSdk.Api.DataModel.Account>) (at Assets/VRoidSDK/Examples/Core/Controller/BaseController.cs:16)
VRoidSDK.Examples.CharacterModelExample.Controller.CharacterModelDetailController:UseCharacterModel (System.Action`1<VRoidSDK.Examples.Core.Renderer.IRenderer>) (at Assets/VRoidSDK/Examples/CharacterModelExample/Scripts/Controller/CharacterModelDetailController.cs:52)
`

**Comment by nyamadan** [comment]
Cache file may be broken. please delete your [Application.temporaryCachePath](https://docs.unity3d.com/2021.2/Documentation/ScriptReference/Application-temporaryCachePath.html) directory.

This error may occur if the app password has been changed.
